libs/common-components/src/lib/selection.service.ts

```diff
@@ -14,7 +14,10 @@ export class SelectionService {
   }
 
   captureSelection() {
-    this.selection = window.getSelection().getRangeAt(0);
+    const selection = window.getSelection();
+    if (selection.rangeCount > 0) {
```

Comment:
Really interesting place to discuss.

Reply:
It was added because of different behavior in the Editor and Viewer during file dropping.